composer.json

```diff
@@ -69,6 +73,12 @@
             "only": ["goalgorilla/open_social", "drupal/social"]
         }
     },
+    "scripts": {
+        "refresh": [
+            "rm -rf composer.lock vendor html/core html/modules/contrib html/profiles/contrib html/themes/contrib",
+            "@composer update -W --ansi"
+        ]
+    },
```

Comment:
@navneet0693 Perhaps we can remove this since it's a command waiting for disaster for developers who do not grasp exactly what it does.

Your working copy of Open Social and themes will both be in `html/*/contrib` and so this command will also throw those away along with your local copy of git. This means that if you have unpushed work or unstaged changes, those changes will be gone.